static/.htaccess

```diff
@@ -14,12 +14,7 @@ RewriteRule ^docs(/(8.0|next))?/components/modeler/desktop-modeler/element-templ
 # Added Self-Managed Modeler redirect
 RewriteRule ^docs/next/self-managed/web-modeler/installation/(.*)$ /docs/next/self-managed/modeler/web-modeler/installation/$1 [R=301,L]
 
-RewriteRule ^docs/apis-clients/working-with-apis-clients/?$ /docs/apis-clients/working-with-apis-tools/ [R=301,L]
 RewriteRule ^docs/next/apis-clients/working-with-apis-clients/?$ /docs/apis-clients/working-with-apis-tools/ [R=301,L]
```

Comment:
Oh! Not sure if you also noticed, but the remaining rule is incorrect on the right side -- it should go to `/docs/next/apis-clients...`, not `/docs/apis-clients...`, because that new URL doesn't exist yet in the current version 😅

Reply:
THANK YOU! 🥇